client/example/src/App.tsx

```diff
@@ -25,29 +43,113 @@ class AppMain extends React.Component<AppMainProps, AppMainStates> {
 
   render() {
     console.log('new App created with uuid: ', this.uuid);
+
+    const x = ndarray(new Float32Array([1, 2, 3, 4, 6, 10])) as NDT;
+    const y = ndarray(new Float32Array([1, 4, 9, 16, 36, 100])) as NDT;
+    const lineProps = {
+      selections: [] as SelectionBase[],
+      plotConfig: {
+        title: 'Sample Line Plot',
+        xLabel: 'x-axis',
+        yLabel: 'y-axis',
+      } as PlotConfig,
+      lineData: [
+        {
+          key: 'squares',
+          lineParams: {
+            colour: 'purple',
+            pointSize: 6,
+            lineOn: true,
+          } as LineParams,
+          x,
+          xDomain: [1, 10],
+          y,
+          yDomain: [1, 100],
+          defaultIndices: false,
+        } as LineData,
+      ],
+      xDomain: [0, 11],
+      yDomain: [0, 101],
+    } as LinePlotProps;
+
+    const values = ndarray(
+      new Float32Array([5, 10, 15, 1.5, 4.5, 3.5]),
+      [3, 2]
+    ) as NDT;
+    const heatmapProps = {
+      selections: [] as SelectionBase[],
+      plotConfig: {
+        title: 'Sample Heatmap Plot',
+        xLabel: 'x-axis',
+        yLabel: 'y-axis',
+      } as PlotConfig,
+      key: 'Example heatmap',
```

Comment:
Remove `key`

Reply:
I think removing `key` should be in another PR - it's unnecessary and unused for all bar the line plots.

Reply:
Done